src/Auth0/Login/Auth0Service.php

```diff
@@ -47,12 +48,12 @@ public function __construct(
         if (!$sessionStorage instanceof StoreInterface) {
             $sessionStorage = new LaravelSessionStore();
         }
-        if (!$sessionStateHandler instanceof SessionStateHandler) {
-            $sessionStateHandler = new SessionStateHandler($sessionStorage);
+        if (!$stateHandler instanceof SessionStateHandler) {
```

Comment:
Checking for the wrong class here ... should be `StateHandler` I believe?

Reply:
Thanks for the heads up!